src/main/java/com/openlattice/shuttle/dates/JavaDateTimeHelper.java

```diff
@@ -1,81 +1,123 @@
                 R result = parseFunction.apply( date, formatter );
                 return postParseFunction.apply( result, datePattern );
-            } catch ( DateTimeParseException e) {
-                if ( datePattern.equals(datePatterns[datePatterns.length - 1]) ) {
-                    logger.error("Unable to parse date {}, please see debug log for additional information: {}.", date,e);
+            } catch ( DateTimeParseException e ) {
+                if ( datePattern.equals( datePatterns[ datePatterns.length - 1 ] ) ) {
+                    logger.error( "Unable to parse date {}, please see debug log for additional information: {}.",
+                            date,
+                            e );
                 }
             } catch ( ExecutionException ex ) {
-                logger.error("ExecutionException loading pattern from cache", ex);
+                logger.error( "ExecutionException loading pattern from cache", ex );
             }
+
         }
         return null;
     }
 
-    public LocalDate parseDate(String date) {
+    public LocalDate parseDate( String date ) {
         return parseWithTwoDigitYearHandling( date, LocalDate::parse, ( ld, datePattern ) -> {
-            if (  checkDatePatternIsTwoDigitYear( datePattern ) ) {
+            if ( checkDatePatternIsTwoDigitYear( datePattern ) ) {
                 // TODO: break this out into its own transform that specifies the date boundaries for two-digit years
-                if ((ld.getYear() - LocalDate.now().getYear()) > 20) {
-                    ld = ld.withYear(ld.getYear() - 100);
+                if ( ( ld.getYear() - LocalDate.now().getYear() ) > 20 ) {
+                    ld = ld.withYear( ld.getYear() - 100 );
                 }
             }
             return ld;
-        });
+        } );
     }
 
-    public OffsetDateTime parseDateTime(String date) {
-        return parseWithTwoDigitYearHandling( date, ( toParse, formatter ) -> {
-            LocalDateTime parsed = LocalDateTime.parse( toParse, formatter );
-            return parsed.atZone( tz.toZoneId() ).toOffsetDateTime();
-        }, ( odt, datePattern ) -> {
-            if (  checkDatePatternIsTwoDigitYear( datePattern ) ) {
-                // TODO: break this out into its own transform that specifies the date boundaries for two-digit years
-                if ((odt.getYear() - LocalDate.now().getYear()) > 20) {
-                    odt = odt.withYear(odt.getYear() - 100);
-                }
+    public OffsetDateTime parseDateTime( String date ) {
+        try {
+            // DateTime: first try parsing as iso-string into OffsetDateTime
+            // TimeZone: if Timezone is not consistent with reported TimeZone: warning
+            OffsetDateTime odt = OffsetDateTime.parse( date );
+            if ( tz.isPresent() ) {
+                if ( tz.orElse( Constants.DEFAULT_TIMEZONE ).toZoneId() == odt.getOffset() );
```

Comment:
I think there should be braces after this if statement and not a semicolon. Right now it's an empty statement